Base/usr/share/man/man8/umount.md

```diff
@@ -0,0 +1,19 @@
+## Name
+
+umount - unmount file system
+
+## Synopsis
+
+```**sh
+$ umount <mountpoint>
+```
+
+## Description
+
+`umount` run as root will unmount a mount point specified in argument.
+
+## Examples
+
+```sh
+# umount
```

Comment:
The `umount` command does nothing without arguments.

Reply:
Fixed.